Tweeter/Models/Twit.cs

```diff
@@ -8,6 +8,7 @@ namespace Tweeter.Models
     public class Twit
     {
         public int TwitId { get; set; }
+        public string TwitName { get; set; }
```

Comment:
you should get the `Username` property off the ApplicationUser model (since a Twit is associated with an ApplicationUser, you have access to it)